README-windows.md

```diff
@@ -28,179 +47,105 @@ Server:
+& $Env:ProgramFiles\docker\dockerd.exe --register-service --experimental
 
+# allow Docker CLI commands to be run from any command line
+[Environment]::SetEnvironmentVariable("Path", "${Env:Path};${Env:ProgramFiles}\docker", [EnvironmentVariableTarget]::Machine)
 ```
-> docker version
 
-Client:
- Version:           18.06.1-ce
- API version:       1.38
- Go version:        go1.10.3
- Git commit:        e68fc7a
- Built:             Tue Aug 21 17:21:31 2018
- OS/Arch:           darwin/amd64
- Experimental:      false
+## Build the LCOW kernel
 
-Server:
- Engine:
-  Version:          18.06.1-ce
-  API version:      1.38 (minimum version 1.12)
-  Go version:       go1.10.3
-  Git commit:       e68fc7a
-  Built:            Tue Aug 21 17:28:38 2018
-  OS/Arch:          linux/amd64
-  Experimental:     false
-
-> linuxkit version
+To run Linux containers with LCOW requires a lightweight kernel image that runs on the Hyper-V subsystem. At this time, the
+[last published release 4.14.29-0aea33bc](https://github.com/linuxkit/lcow/releases) of the LCOW kernel image is outdated and includes different artifacts than what the current Docker installation expects. Docker expects the LCOW kernel image files to be named `initrd.img` and `kernel` and to be installed to `$Env:Program Files\Linux Containers`.
 
-linuxkit version v0.6+
-commit: c8449ba2dbc0e26a80b2e4acfb4946be68d3239b
+Therefore, a new kernel image must be built using the [linuxkit](https://github.com/linuxkit/linuxkit) tooling. Additionally, building from source requires `git`, `make` and `docker` itself.
 
-> make --version
-
-GNU Make 3.81
-Copyright (C) 2006  Free Software Foundation, Inc.
-This is free software; see the source for copying conditions.
-There is NO warranty; not even for MERCHANTABILITY or FITNESS FOR A
-PARTICULAR PURPOSE.
-
-This program built for i386-apple-darwin11.3.0
-```
+### Building on Windows
 
-#### Alternatively, building on Windows
+Windows can be used to build the kernel image.
 
-Alternatively, Windows can be used to build this VM, with the help of Chocolatey to install prerequisites.
+Run the following PowerShell script to:
 
-* Install Chocolatey
+* Install Chocolatey package manager for Windows
+* Install build tooling with Chocolatey
+* Copy and build LinuxKit source
+* Copy and build LCOW kernel image source
+* Deploy LCOW kernel image to host
 
 ```powershell
+# install Chocolatey
 Set-ExecutionPolicy Bypass -Scope Process -Force; iex ((New-Object System.Net.WebClient).DownloadString('https://chocolatey.org/install.ps1'))
-```
-
-* Install Tooling
 
-```powershell
+# install build tooling
 choco install -y git golang make
-```
-
-* Acquire and build LinuxKit
 
-For Chocolatey tools to be available on PATH, close and reopen a new terminal. The `docker` CLI tool should already be
-available from prior steps installing Docker. Acquire the Go source and build the tool with `go get`:
+# make build tools available in current session
+refreshenv
```

Comment:
These simplified / condensed steps don't quite work right. I'm tweaking some things to make sure they do